WordPress/Classes/ViewRelated/Blog/Blog Dashboard/DashboardCard.swift

```diff
@@ -150,9 +153,22 @@ enum DashboardCard: String, CaseIterable {
             return DashboardJetpackSocialCardCell.shouldShowCard(for: blog)
         case .googleDomains:
             return FeatureFlag.domainFocus.enabled && isJetpack
+        case .dynamic:
+            return false
         }
     }
 
+    func shouldShow(
+        for blog: Blog,
+        dynamicCardPayload: DashboardDynamicCardModel.Payload,
+        isJetpack: Bool = AppConfiguration.isJetpack
```

Comment:
Do we need to pass `isJetpack` in? I assume we'll always pass in `AppConfiguration.isJetpack`. Is it for testability?

Reply:
@alpavanoglu  That's correct, static methods or properties called within a function implementation are difficult to mock. And I also inspired from the other `shouldShow` method in `DashboardCard` enum.